source/tutorials/write-update-documents.txt

```diff
@@ -216,14 +210,11 @@ value in these documents to ``"Manhattan (north)"``. Because the ``upsert`` opti
 set to ``true``, the {+driver-short+} inserts a new document if the query filter doesn't 
 match any existing documents.
 
-
 .. code-block:: java
    :copyable: true
 
-    ObservableSubscriber <UpdateResult> updateManySubscriber = new SubscriberHelpers.OperationSubscriber<>();
-    restaurants.updateMany(eq("borough", "Manhattan"), set("borough", "Manhattan
-    (north)"), new UpdateOptions().upsert(true)).subscribe(updateManySubscriber);
-    updateManySubscriber.await();
+    Mono.from(restaurants.updateMany(eq("borough", "Manhattan"), set("borough",
+    "Manhattan (north)"), new UpdateOptions().upsert(true))).block();
```

Comment:
Indent the second line of code to make it clear that it's a continutation of the method opened in the first line